### DISCUSSION TITLE
thaoluan

### DISCUSSION THREAD

**Discussion by Khaidtraiii** [Khaidtraiii/Nhom_3_10A13#2] [Announcements]
.

**Comment by Khaidtraiii** [comment]
alooo

**Reply by Khaidtraiii** [reply]
alooo

**Reply by Khaidtraiii** [reply]
eeee ai biết mail thầy ko add đi

**Reply by Khaidtraiii** [reply]
ê mng t giao việc rồi á trong project á

**Reply by Btranne** [reply]
Ủa

**Reply by Btranne** [reply]
Ủa

**Reply by Khaidtraiii** [reply]
có ý tưởng gì không mng

**Reply by Btranne** [reply]
T kh thấy project :))

**Reply by Khaidtraiii** [reply]
câu đầu dùng thơ có sẵn xong mấy câu tự chế không mng?? vd: cảnh nào cảnh chẳng đeo sầu...........; chiều chiều đứng ra đứng ngõ sau........

**Reply by Khaidtraiii** [reply]
project có phần beta với phần cũ á
bấm cái ở dưới

**Reply by Btranne** [reply]
Ủa r làm mấy câu thơ z :))

**Reply by Ngtkim06** [reply]
ủa sao outlook k thông báo tụi m nhắn gì luôn

**Reply by PhucKhangg** [reply]
Kay chiêng Văn sáng tác vài câu chơi

**Reply by Sanly1111** [reply]
hehehe

**Reply by Sanly1111** [reply]
**chao mng**

**Reply by Khaidtraiii** [reply]
alo mng t chia việc nhâ

**Reply by Ngtkim06** [reply]
ok chia đe

**Reply by Khaidtraiii** [reply]
vào project tuan 2

**Reply by Btranne** [reply]
Ủa thơ là t tìm thơ 1-2 câu hay là nguyên bài thơ dài z

**Reply by Khaidtraiii** [reply]
muốn lấy nhiêu thì lấy

**Reply by Khaidtraiii** [reply]
lấy xong để trong issue cho khỏi trôi

**Reply by PhucKhangg** [reply]
Ê tìm xong bìa rồi á xem được chưa nếu chuaq được thì để t tìm cái khác

**Reply by PhucKhangg** [reply]
Vào code xem

**Reply by Khaidtraiii** [reply]
ê san với trân thơ ca gì đâu hết rồi

**Reply by Btranne** [reply]
Ủa hạn khi nào z

**Reply by Btranne** [reply]
Tại qua giờ t bận quá chưa có thgian làm :))

**Reply by Btranne** [reply]
Ủa mà phân tích thơ là làm nguyên bài văn luôn hay ý thôi ?

**Reply by PhucKhangg** [reply]
Hạn là T7 hqua hay sao á

**Reply by Sanly1111** [reply]
Ủa bây giờ cái bài phân tích dài quá đưa vô project ko đủ thì đem vô đâu vậy @Khaidtraiii

**Reply by Sanly1111** [reply]
à thôi t mới tìm đc rồi :)))

**Reply by Khaidtraiii** [reply]
@Sanly1111 alo alo đừng lụm văn mẫu nhaa

**Reply by Khaidtraiii** [reply]
nêu cảm nghĩ của mình thôi

**Reply by Khaidtraiii** [reply]
chứ dài quá mà mấy cái đó sgk hay mạng có đầy thì ai thèm đọc :))
